packages/big-design/src/utils/localizationProvider/localizationProvider.ts

```diff
@@ -21,5 +39,23 @@ export const createLocalizationProvider = (locale: string) => {
     monthsLong,
     // Required by datepicker.
     formatLong: {},
+    formatTime: timeFormatter.format,
   };
 };
+
+export function getTimeIntervals(locale: string) {
+  const localization = createLocalizationProvider(locale);
```

Comment:
I wonder if we want to create the localizationProvider inside the component like we do with `DatePicker`? 

Down the line we eventually would want to be able to pass a localization provider into the components so that we are not duplicating efforts.

Reply:
Sounds good. But I would defer this for now till we tackle `DateTimePicker`, which does combine the two components. As part of that work we can work out how to abstract localization provider.

Reply:
I'd like us to follow suite with what we are doing with `DatePicker` so that it'll be a uniform refactor when we tackle `DateTimePicker`.